src/components/expanded-state/chart/ChartExpandedStateHeader.js

```diff
@@ -78,10 +78,10 @@ export default function ChartExpandedStateHeader({
   const priceSharedValue = useSharedValue('');
 
   useEffect(() => {
```

Comment:
shouldn't we use something `useImmediateEffect` here?

Reply:
Probably, but I don't think it should be part of this PR